atoMEC/staticKS.py

```diff
@@ -194,9 +195,17 @@ def make_lbound(eigvals):
 
         for l in range(config.lmax):
             lbound_mat[:, l] = (2.0 / config.spindims) * np.where(
-                eigvals[:, l] < 0, 2 * l + 1.0, 0.0
+                eigvals[:, l] < 0.0, 2 * l + 1.0, 0.0
             )
 
+        # force bound levels if there are convergence issues
+        if config.force_bound != []:
```

Comment:
Empty containers evaluate to `False` in Python. You can just write:
```suggestion
        if config.force_bound:
```
to check for a non empty container.